src/platforms/esp32/components/avm_sys/platform_nifs.c

```diff
@@ -628,6 +629,285 @@ static term nif_crypto_hash(Context *ctx, int argc, term argv[])
     return term_from_literal_binary(digest, digest_len, &ctx->heap, ctx->global);
 }
 
+static const AtomStringIntPair cipher_no_iv_table[] = {
+    { ATOM_STR("\xB", "aes_128_ecb"), MBEDTLS_CIPHER_AES_128_ECB },
+    { ATOM_STR("\xB", "aes_192_ecb"), MBEDTLS_CIPHER_AES_192_ECB },
+    { ATOM_STR("\xB", "aes_256_ecb"), MBEDTLS_CIPHER_AES_256_ECB },
+    SELECT_INT_DEFAULT(MBEDTLS_CIPHER_NONE)
+};
+
+static const AtomStringIntPair cipher_iv_table[] = {
+    { ATOM_STR("\xB", "aes_128_cbc"), MBEDTLS_CIPHER_AES_128_CBC },
+    { ATOM_STR("\xB", "aes_192_cbc"), MBEDTLS_CIPHER_AES_192_CBC },
+    { ATOM_STR("\xB", "aes_256_cbc"), MBEDTLS_CIPHER_AES_256_CBC },
+    { ATOM_STR("\xE", "aes_128_cfb128"), MBEDTLS_CIPHER_AES_128_CFB128 },
+    { ATOM_STR("\xE", "aes_192_cfb128"), MBEDTLS_CIPHER_AES_192_CFB128 },
+    { ATOM_STR("\xE", "aes_256_cfb128"), MBEDTLS_CIPHER_AES_256_CFB128 },
+    { ATOM_STR("\xB", "aes_128_ctr"), MBEDTLS_CIPHER_AES_128_CTR },
+    { ATOM_STR("\xB", "aes_192_ctr"), MBEDTLS_CIPHER_AES_192_CTR },
+    { ATOM_STR("\xB", "aes_256_ctr"), MBEDTLS_CIPHER_AES_256_CTR },
+    SELECT_INT_DEFAULT(MBEDTLS_CIPHER_NONE)
+};
+
+static const AtomStringIntPair padding_table[] = {
+    { ATOM_STR("\x4", "none"), MBEDTLS_PADDING_NONE },
+    { ATOM_STR("\xC", "pkcs_padding"), MBEDTLS_PADDING_PKCS7 },
+    SELECT_INT_DEFAULT(-1)
```

Comment:
Is it known that neither of the above `_PADDING_` enums don’t have a value of -1?